setup.py

```diff
@@ -62,6 +62,8 @@
 
 version_path = Path("charmcraft/version.py")
 version_backup = Path("charmcraft/version.py~")
+if version_backup.exists():
+    version_backup.unlink()
```

Comment:
It could be interesting to add a comment explaining why we are doing this, otherwise someone could try to remove these lines in a future refactoring.